src/components/FullHeader.tsx

```diff
@@ -3,18 +3,18 @@ import { IconButton } from "./IconButton";
 import { HeaderTitle } from "./HeaderTitle";
 
 type FullHeader = {
-  classes: string;
-  name: string;
+	classes: string;
+	name: string;
 };
 
 export function FullHeader(props: FullHeader) {
-  return (
-    <div className="flex items-center gap-24">
-      <HeaderTitle
-        classes="text-title text-primary-default-Solid "
-        name={""}
-      ></HeaderTitle>
-      <IconButton classes="w-12 h-12"></IconButton>
-    </div>
-  );
+	return (
+		<div className="flex items-center gap-24">
+			<HeaderTitle
+				classes="text-title text-4xl text-primary-default-Solid "
+				name={""}
```

Comment:
empty name here

Reply:
if this is a property, give the property in HeaderTitle a default value to remove the error